Package.swift

```diff
@@ -30,7 +30,8 @@ let package = Package(
     targets: [
         .target(name: "libsocket", dependencies: []),
         .target(name: "libreprl", dependencies: []),
-        .target(name: "libcoverage", dependencies: [], linkerSettings: [.linkedLibrary("rt", .when(platforms: [.linux]))]),
+        // TODO: Verify that -O3 is actually doing something
```

Comment:
Probably not, since the swift build system should use it automatically when using `-c release`, but feel free to double check

Reply:
Here's what I got with `swift build -c release -v`, with `-O3` included in `Package.swift`

`/home/b/tools/swift-5.2.5-RELEASE-ubuntu20.04/usr/bin/clang -target x86_64-unknown-linux-gnu -fPIC -g -O2 -DSWIFT_PACKAGE=1 -fblocks -fmodules -fmodule-name=libcoverage -I /home/b/projects/fuzzilli/Sources/libcoverage/include -fmodules-cache-path=/home/b/projects/fuzzilli/.build/x86_64-unknown-linux-gnu/release/ModuleCache -O3 -MD -MT dependencies -MF /home/b/projects/fuzzilli/.build/x86_64-unknown-linux-gnu/release/libcoverage.build/coverage.c.d -c /home/b/projects/fuzzilli/Sources/libcoverage/coverage.c -o /home/b/projects/fuzzilli/.build/x86_64-unknown-linux-gnu/release/libcoverage.build/coverage.c.o`

It's a little weird having both `-O2` and `-O3`, but the performance gain seems preferable.